cypress/support/commands.ts

```diff
@@ -128,6 +128,12 @@ Cypress.Commands.add("saveToDatabase", () => {
 });
 
 Cypress.Commands.add("enterEditMode", () => {
+  // TODO: uncomment and check if working after TASK-180 would be merged
+  // const actualRevisionValue = "actual";
+  // cy.get("[data-cy=revision-select]")
+  //   .select(RevisionTypeLabels[actualRevisionValue])
+  //   .should("have.value", actualRevisionValue);
+  // cy.get("[data-cy=revision-select]").blur();
```

Comment:
Mam jedną teorię dlaczego to nie działa i myślę, że to powinny naprawić zmiany które dodaje w swoim tasku.
Na razie zakomentowałem to, zakomentowałem to przełączenie miesiąca, ponieważ zbyt długo nad tym siedziałem i doszedłem do wniosku, że lepiej będzie jeśli po prostu sprawdzę czy to zadziała po zmergowaniu mojego taska. 

Jeśli nie, to będę dalej z tym walczyć